README.md

```diff
@@ -1 +1,29 @@
-# Operator Marketplace
+# Marketplace Operator
+
+## Project Status: pre-alpha
+
+## Prerequisites
+In order to deploy the Marketplace Operator, you must:
+1. Have an OpenShift cluster with Operator Lifecycle Manager [installed](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/install/install.md).
+2. Be logged in as a user with the RBAC permissions required to deploy this operator. These RBAC permissions are currently undefined, so log in as a user with the Cluster Admin Role.
+
+## Making changes to the Marketplace Operator
+The Marketplace Operator is hosted at `quay.io/redhat/marketplace-operator` and not all developers have pull and push privileges on this image. If you do not have these privileges and are developing new features for the Marketplace Operator you must build and push your image to a registry where you have said privileges. The steps below outline said process:
```

Comment:
Are these instructions for creating a marketplace operator from scratch, or a workaround for developers without pull privileges?

Reply:
Technically both? You do build a marketplace operator from scratch locally and said image is needed to be pushed to a registry for the workaround. I updated the section to be clearer, let me know your thoughts.